src/app/reset-password/page.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import supabase from '../../api/supabase/createClient';
+
+export default function ResetPassword() {
+    const [newPassword, setPassword] = useState('');
+    const { push } = useRouter();
+    const resetPassword = async () => {
+        await supabase.auth.updateUser({ password: newPassword});
+        push("/login");
+    }
```

Comment:
Add error handling similar to how you did for `handleSignUp`.

Reply:
gotcha!